src/textual/_compositor.py

```diff
@@ -800,41 +806,83 @@ def _get_renders(
                     _Region(delta_x, delta_y, new_width, new_height)
                 )
 
-    def render(self, full: bool = False) -> RenderableType | None:
-        """Render a layout.
+    def render_update(
+        self, full: bool = False, screen_stack: list[Screen] | None = None
+    ) -> RenderableType | None:
+        """Render an update renderable
+
+        Args:
+            full: Enable full update, or `False` for a partial update.
 
         Returns:
-            A renderable
+            A renderable for the update, or `None` if no update was required.
         """
 
-        width, height = self.size
-        screen_region = Region(0, 0, width, height)
-
-        if full:
-            update_regions: set[Region] = set()
+        visible_screen_stack.set(screen_stack or [])
```

Comment:
If `screen_stack` is `[]`, then this evaluates to the right-hand empty list `[]`.
So, if later someone mutates the `screen_stack` that was `[]`, the change will not be reflected in `visible_screen_stack` because we have a reference to a different list.

Can this be an issue in any way?

Reply:
I don't think it will be an issue, but it can't hurt to be explicit with `None`...